assets/js/pages/Users/UserForm.stories.jsx

```diff
@@ -35,18 +41,24 @@ export default {
         type: 'text',
       },
     },
-    password: {
-      description: 'Password',
+    createdAt: {
+      description: 'User creation time',
       control: {
         type: 'text',
       },
     },
-    confirmPassword: {
-      description: 'Password confirmation',
+    udpatedAt: {
+      description: 'User last edition time',
```

Comment:
Just an idea: 
Timestamp of last user data update

Reply:
I will just leave with `User last update timestamp` to be more aligned with the `createdAt: 'User creation timestamp'`